libsycl/include/sycl/detail/config.hpp

```diff
@@ -0,0 +1,64 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+///
+/// \file
+/// This file contains the declaration of the macros defining attributes for
+/// exported methods and defining API namespaces.
+///
+//===----------------------------------------------------------------------===//
+
+#ifndef __LIBSYCL_DETAIL_CONFIG_HPP
```

Comment:
Is there a reason you use double underscores everywhere? That style tends to be use by compiler-defined stuff, so we avoid it in libc++.

Reply:
https://github.com/llvm/llvm-project/pull/144372/commits/fdf4ddb4eefcd54c7b56db9b100dc491e4cea128